com.valvesoftware.Steam.CompatibilityTool.Proton.metainfo.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.valvesoftware.Steam.CompatibilityTool.Proton</id>
+  <extends>com.valvesoftware.Steam</extends>
+  <name>Proton</name>
+  <summary>Compatibility tool for Steam Play based on Wine and additional components</summary>
```

Comment:
Add metainfo translation to pt_BR and es for Proton
```suggestion
  <summary>Compatibility tool for Steam Play based on Wine and additional components</summary>
  <summary xml:lang="pt_BR">Ferramenta de compatibilidade para Steam Play baseada no Wine e componentes adicionais</summary>
  <summary xml:lang="es">Herramienta de compatibilidad para Steam Play basada en Wine y componentes adicionales</summary>
```

Reply:
Hm, I guess the summary should indicate that it isn't official. Something like
>Compatibility tool based on Wine (unofficial flatpak build)

Reply:
Alternatively community build. The important thing to stress ia this isn't backed by Valve. This should also be made obvious when selecting tool within Steam

Reply:
I might even go as far as name "Proton (community build)

Reply:
So what should `name` and `summary` be like, exactly? My English is too limited to word it nicely.

Reply:
Looks like for the app this stuff is in description https://github.com/flathub/com.valvesoftware.Steam/blob/beta/com.valvesoftware.Steam.metainfo.xml#L12

Reply:
I've set tool's name to `Proton (community build)` both in metainfo and compatibilitytool.vdf. Let's hope it won't be too long for UI parts.